index.js

```diff
@@ -246,6 +246,22 @@ export async function setupPlugin({ config, global, jobs }) {
     })
 }
 
+function getElementByOrderZero(json) {
+  //console.log("here ok");
+  if (!json.elements || !Array.isArray(json.elements)) {
+    return null; // or whatever default value you'd like to return
+  }
+  const elements = json.elements;
+    
+  if (elements.length < 1){
+    return null;
+  }
+  for (let i = 0; i < elements.length; i++) {
+    if (elements[i].order === 0) {
+      return elements[i];
+    }
+  }
```

Comment:
Could be written much more succinctly 
```suggestion
  return json.elements.find(x => x.order === 0) || null
```